### ISSUE TITLE
README examples

### ISSUE THREAD

**Issue by alexchandel** [o11c/rust-xdg#7]
How about a simple example in the README for creating a data folder, a cache folder, and a config folder? It should demonstrate how easy it is.

**Comment by o11c** [comment]
Yeah, I probably should do that. I've been putting off a lot of stuff because I'm still not sure what it should be.

I have a vague plan forming that involves traits though.

**Comment by alexchandel** [comment]
Could be nice, but in the meantime having something in the Readme, describing the current state, is paramount. Otherwise people just click away!
